binding/nodejs/test/index.test.ts

```diff
@@ -0,0 +1,133 @@
+//
+// Copyright 2024 Picovoice Inc.
+//
+// You may not use this file except in compliance with the license. A copy of the license is located in the "LICENSE"
+// file accompanying this source.
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+// specific language governing permissions and limitations under the License.
+//
+'use strict';
+
+import { Cobra, CobraErrors } from '../src';
+import * as fs from 'fs';
+import { WaveFile } from 'wavefile';
+
+import { getSystemLibraryPath } from '../src/platforms';
+
+import { getAudioFile } from './test_utils';
+
+const WAV_PATH = 'sample.wav';
+const EXPECTED_VOICE_PROBABILITY_SCORES = [
+  0.00350041501224041, 0.017226237803697586, 0.022328782826662064,
+  0.024964014068245888, 0.017900168895721436, 0.028114058077335358,
+  0.1203354075551033, 0.11426965147256851, 0.0626743733882904,
+  0.028843538835644722, 0.9569169282913208, 0.9879112839698792,
+  0.9837538003921509, 0.9286680221557617, 0.9479621648788452,
+  0.9499486684799194, 0.9634374976158142, 0.9786311388015747,
+  0.9836912155151367, 0.9827406406402588, 0.9849273562431335,
+  0.9784052968025208, 0.9812748432159424, 0.9742639660835266,
+  0.9664738178253174, 0.9518184661865234, 0.9407353401184082, 0.814495861530304,
+  0.118283212184906, 0.01206541433930397, 0.014447852969169617,
+  0.015830140560865402, 0.012734980322420597, 0.01062167901545763,
+  0.010828903876245022, 0.00736008258536458, 0.014871003106236458,
+  0.00925009697675705, 0.009878940880298615, 0.008195844478905201,
+  0.008736720308661461, 0.01084984466433525,
+];
```

Comment:
It's not a good idea to get this detailed with the test expectations. Look at how the other tests do it:
e.g.) https://github.com/Picovoice/cobra/blob/main/binding/android/CobraTestApp/cobra-test-app/src/androidTest/java/ai/picovoice/cobra/testapp/CobraTest.java

They use a loss metric that gives a bit more breathing room for variation.